docs/.dockerignore

```diff
@@ -0,0 +1,12 @@
+# We cannot call `docker build` out of top dir since this is controlled by
```

Comment:
I don't get how is this related to this dockerfile

Reply:
Will remove the comment and add sections per ignore.